README.md

```diff
@@ -74,8 +74,93 @@ require_once __DIR__ . DIRECTORY_SEPARATOR . 'vendor' . DIRECTORY_SEPARATOR . 'a
 $runner->addExtension(new \mageekguy\atoum\blackfire\extension());
 ```
 
+## Other examples
+
+### Define custom metrics
+
+```php
+$this
+    ->blackfire()
+        ->defineMetric(new \Blackfire\Profile\Metric("example_method_calls", "=Example::method"))
```

Comment:
Because, that's how we pass a custom metric to the profile configuration.

It must be an instance of a `Blackfire\Profile\Metric object` : https://github.com/blackfireio/php-sdk/blob/master/src/Blackfire/Profile/Configuration.php#L162